packages/flutter/test/rendering/viewport_test.dart

```diff
@@ -979,4 +980,163 @@ void main() {
+          '   situation typically happens when a scrollable widget is nested\n'
+          '   inside another scrollable widget.\n'
+          '   If this widget is always nested in a scrollable widget there is\n'
+          '   no need to use a viewport because there will always be enough\n'
+          '   horizontal space for the children. In this case, consider using a\n'
+          '   Row instead. Otherwise, consider using the "shrinkWrap" property\n'
+          '   (or a ShrinkWrappingViewport) to size the width of the viewport\n'
+          '   to the sum of the widths of its children.\n'
+      );
+    });
+
+    testWidgets('Vertical viewport was given unbounded width', (WidgetTester tester) async {
+      await expectFlutterError(
+        widget: buildNestedWidget(Axis.horizontal, Axis.vertical),
+        tester: tester,
+        message:
+          'FlutterError\n'
+          '   Vertical viewport was given unbounded width.\n'
+          '   Viewports expand in the cross axis to fill their container and\n'
+          '   constrain their children to match their extent in the cross axis.\n'
+          '   In this case, a vertical viewport was given an unlimited amount\n'
+          '   of horizontal space in which to expand.\n'
+      );
+    });
+
+    testWidgets('Vertical viewport was given unbounded height', (WidgetTester tester) async {
+      await expectFlutterError(
+        widget: buildNestedWidget(Axis.vertical, Axis.vertical),
+        tester: tester,
+        message:
+          'FlutterError\n'
+          '   Vertical viewport was given unbounded height.\n'
+          '   Viewports expand in the scrolling direction to fill their\n'
+          '   container. In this case, a vertical viewport was given an\n'
+          '   unlimited amount of vertical space in which to expand. This\n'
+          '   situation typically happens when a scrollable widget is nested\n'
+          '   inside another scrollable widget.\n'
+          '   If this widget is always nested in a scrollable widget there is\n'
+          '   no need to use a viewport because there will always be enough\n'
+          '   vertical space for the children. In this case, consider using a\n'
+          '   Column instead. Otherwise, consider using the "shrinkWrap"\n'
+          '   property (or a ShrinkWrappingViewport) to size the height of the\n'
+          '   viewport to the sum of the heights of its children.\n'
+      );
+    });
+  });
+
+  test('Viewport debugThrowIfNotCheckingIntrinsics() control test', () {
+    final RenderViewport renderViewport = RenderViewport(
+      crossAxisDirection: AxisDirection.right, offset: ViewportOffset.zero()
+    );
+    FlutterError error;
+    try {
+      renderViewport.computeMinIntrinsicHeight(0);
+    } on FlutterError catch (e) {
+      error = e;
+    }
+    expect(error, isNotNull);
+    expect(
+      error.toStringDeep(),
+      'FlutterError\n'
```

Comment:
indent all the lines to match the indentation of 'FlutterError\n'
Sorry IntelliJ doesn't match the Flutter style guide.

Reply:
yes, missed this one, thankyou!